SECURITY.md

```diff
@@ -0,0 +1,49 @@
+## Security Policy
+
+The Wikijump Team prioritizes security and will fix issues as soon as they are disclosed.
+
+If you discover a security vulnerability, please reach out to a member of the Wikijump Security Team, who may be contacted as follows:
+
+| GitHub       | Wikidot                                                               | Discord           | Email                     |
+|--------------|-----------------------------------------------------------------------|-------------------|---------------------------|
+| @ammongit    | [aismallard](http://www.wikidot.com/account/messages#/new/4598089)    | `aismallard#0002` | ammon.i.smith@gmail.com   |
+| @rossjrw     | [Croquembouche](http://www.wikidot.com/account/messages#/new/2893766) |                   | rossjrw@gmail.com         |
+| @stormbreath | [stormbreath](http://www.wikidot.com/account/messages#/new/3075960)   |                   |                           |
+| @danieltharp | [pxdnbluesoul](http://www.wikidot.com/account/messages#/new/1414125)  |                   | daniel.tharp@gmail.com    |
+
+### Wikijump
+
+After receiving a report of a security issue within Wikijump, we will validate and determine its scope.
+Once this is done, we will issue an estimate for the time to completion. When a fix has been merged in and deployed,
+you will be notified privately. If this security issue impacts Wikijump _only_, you may disclose it publicly.
+
+### Wikidot
+
+The Wikijump Security Team also receives reports of security issues within Wikidot. You are encouraged to report
+any vulnerabilities or exploits you discover.
+
+However, standard security disclosure cannot apply here, for three important reasons:
+
+1. Wikidot is controlled by a third-party, so we are unable to apply security patches.
+2. Wikidot is completely unmaintained, so even severe vulnerabilities will not be fixed.
+3. Wikidot is actively used by a large number of users, and public disclosure of a vulnerability has the potential to cause a significant amount of harm with no recourse.
+
+Because of this, the strategy we utilize is (unfortunately) based on _mitigation and concealment_.
+When you make your report, we will document it privately like other vulnerabilities, and assess its level of current impact.
+If any occurrences of the issue are found in the wild, every effort will be made to remove or replace them, while minimizing
+the number of people aware of the issue. If needed, a minimal number of administrators of large Wikidot sites will be made aware
+of recommended mitigations.
+Finally, the potential impact on Wikijump will be determined, and if the issue is also present there, it will be promptly fixed.
+
+As such, you will be asked to not discuss the exploit. This request will only be lifted in the very unlikely scenario that Wikidot releases a patch.
+
+A Wikidot bug will **not** be created, both because it is public and will become known to any potential bad actors, but because the issue is extremely
```

Comment:
Define 'wikidot bug' - I assume e.g. an entry on feedback.wikidot.com et al?

Reply:
Good call, will clarify. Changed in 7f101e8.